cloud_chatops/lib/pr_reminder.py

```diff
@@ -179,7 +181,10 @@ def _github_to_slack_username(self, user: str) -> str:
         :return: Slack ID or GitHub username
         """
         if user not in self.slack_ids:
-            user = self.default_user_id
+            # If the PR author is not in the Slack ID mapping
+            # then we set the user to mention as David Fairbrother
+            # as the team lead to deal with this PR.
+            user = "U01JG0LKU3W"
```

Comment:
Also you've hardcoded this here too - you could again set it as a constant variable at the top of the file 
Or better yet in a .conf file that the program reads in - left for a future PR?

Reply:
Yes that's a good idea